CONTRIBUTORS_GUIDE.md

```diff
@@ -0,0 +1,68 @@
+# Contributors' Guide
+
+This guide will help familiarize contributors to the `containerd/containerd` repository.
+
+## Prerequisite
+
+First read the containerd project's [general guidelines around contribution](https://github.com/containerd/project/blob/main/CONTRIBUTING.md)
+which apply to all containerd projects.
+
+## Getting Started
```

Comment:
The title format is not consistent: some titles only capitalize 1st word and some capitalize every word.

Reply:
Updated